constraints.txt

```diff
@@ -1,4 +1,3 @@
-molecule==2.8.2
-pbr==3.0.1
```

Comment:
Just curious, why don't we need to constrain PBR anymore?

Reply:
Molecule manages its own constraint on pbr. So, this seems like a potential conflict.
https://github.com/metacloud/molecule/blob/master/requirements.txt#L11

Reply:
Ahh.